backend/src/main/java/com/fortunator/api/controller/TransactionController.java

```diff
@@ -0,0 +1,66 @@
+package com.fortunator.api.controller;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fortunator.api.models.Transaction;
+import com.fortunator.api.service.TransactionService;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+@CrossOrigin
+@RestController
+@RequestMapping(value = "/transactions")
+public class TransactionController {
+
+	private static final int SC_BAD_REQUEST = 400;
+	private static final int SC_NO_CONTENT = 204;
+	private static final int SC_OK = 200;
+	private static final int SC_NOT_FOUND = 404;
+	private static final int SC_UNPROCESSABLE_ENTITY = 422;
+	
+	@Autowired
+	private TransactionService transactionService;
+
+	@ApiOperation(value = "Create a new transaction")
+	@ApiResponses(value = { @ApiResponse(code = SC_BAD_REQUEST, message = "One or more fields were filled in incorrectly"),
+			@ApiResponse(code = SC_NOT_FOUND, message = "When not found user or transaction category"),
```

Comment:
Como para criar uma transação precisamos de um `user_id`, se for passado um usuário que não exista, devemos retornar um erro. Porém, não sei se 404 seria o adequado aqui, porque como é um endpoint de transactions, passa a sensação de que a transação não foi encontrada. 

Uma sugestão seria usar o status 403 forbidden, que indica que a requisição foi entendida porém não pode ser prosseguida por quesões de permissão. 

```suggestion
			@ApiResponse(code = SC_NOT_FOUND, message = "When user trying to create transaction does not exist"),
```